index.rst

```diff
@@ -7,652 +7,415 @@ Table of contents
-
-   Dom0 secure updates </developer/services/dom0-secure-updates>
-
-   Qrexec: secure communication across domains </developer/services/qrexec>
-
-   Qrexec v2 (deprecated) </developer/services/qrexec2>
-
-   Qrexec: Qubes RPC internals </developer/services/qrexec-internals>
-
-   Qrexec: Socket-based services </developer/services/qrexec-socket-services>
-
-   Admin API </developer/services/admin-api>
-
-   Qfileexchgd (deprecated) </developer/services/qfileexchgd>
+   developer/services/qfilecopy
+   developer/services/qmemman
+   developer/services/disposablevm-implementation
+   developer/services/dom0-secure-updates
+   developer/services/qrexec
+   developer/services/qrexec2
+   developer/services/qrexec-internals
+   developer/services/qrexec-socket-services
+   developer/services/admin-api
+   developer/services/qfileexchgd
 
 
-
-Debugging
-^^^^^^^^^
-
+.. _developers-debugging:
 
 .. toctree::
    :maxdepth: 1
+   :caption: Developers - debugging
 
-   How to set up a test bench </developer/debugging/test-bench>
-
-   Automated tests </developer/debugging/automated-tests>
-
-   Python profiling <>
-
-   Qube configuration interface </developer/debugging/vm-interface>
-
-   Windows debugging </developer/debugging/windows-debugging>
-
-   Safe remote dom0 terminals </developer/debugging/safe-remote-ttys>
-
-   How to mount LVM images </developer/debugging/mount-lvm-image>
+   developer/debugging/test-bench
+   developer/debugging/automated-tests
+   developer/debugging/vm-interface
+   developer/debugging/windows-debugging
+   developer/debugging/safe-remote-ttys
+   developer/debugging/mount-lvm-image
 
 
-
-Building
-^^^^^^^^
-
+.. _developers-building:
 
 .. toctree::
    :maxdepth: 1
+   :caption: Developers - building
 
-   Qubes builder </developer/building/qubes-builder>
-
-   Qubes builder details </developer/building/qubes-builder-details>
-
-   Development workflow </developer/building/development-workflow>
-
-   Qubes ISO building </developer/building/qubes-iso-building>
-
+   developer/building/qubes-builder
+   developer/building/qubes-builder-details
+   developer/building/development-workflow
+   developer/building/qubes-iso-building
    Qubes template configs <https://github.com/QubesOS/qubes-template-configs>
 
 
-
-Releases
-^^^^^^^^
-
+.. _developers-releases:
 
 .. toctree::
    :maxdepth: 1
+   :caption: Developers - releases
 
-   Release notes </developer/releases/notes>
-
-   Release schedules </developer/releases/schedules>
-
-   Release checklist </developer/releases/todo>
-
-   Version scheme </developer/releases/version-scheme>
-
+   developer/releases/notes
+   developer/releases/schedules
+   developer/releases/todo
+   developer/releases/version-scheme
 
 
 External Documentation
 ^^^^^^^^^^^^^^^^^^^^^^
 
-
 Unofficial, third-party documentation from the Qubes community and
 others.
 
-Operating System Guides
-^^^^^^^^^^^^^^^^^^^^^^^
-
```

Comment:
The same for the subsection to the External Documentation as in the comment about the Developer Documentaion